Documentation/building/linux-instructions.md

```diff
@@ -158,3 +158,63 @@ Finally, let's build coreclr with updated clang/llvm. If you meet a lldb related
 lgs@ubuntu time ROOTFS_DIR=/work/dotnet/rootfs-coreclr/arm ./build.sh arm release clean cross 
 ```
 
+Additional optimization levels for ARM/Linux: -Oz and -Ofast
+============================================================
+
+This instruction is to enable additional optimization levels such as -Oz and -Ofast on ARM/Linux. The below table shows what we have to enable for the code optimization of the CoreCLR run-time either the size or speed on embedded devices. 
+
+| **Content** | **Build Mode** | **Clang/LLVM (Linux)** |
+| --- | --- | --- |
```

Comment:
@jkotas I re-submitted new PR after applying your comments. Could you review this PR?